### ISSUE TITLE
Space missing when CSS disabled on OD TV 

### ISSUE THREAD

**Issue by aeroplanejane** [bbc/simorgh#6906]
**Describe the bug**
When CSS is disabled on OD TV, the title and the datestamp run together. There should be a space between the words ‘Caalamka’ and ‘12’
**To Reproduce**
1. Go to https://www.test.bbc.com/somali/bbc_somali_tv/tv_programmes/w13xttqt 
2. Disable CSS 
3. Note the h2 below the player 

**Expected behaviour**
A space between the words ‘Caalamka’ and ‘12’

**Screenshots**

<img width="325" alt="Screenshot 2020-06-15 at 12 11 24" src="https://user-images.githubusercontent.com/22519682/84652343-61469600-af03-11ea-8f4b-b1e9172e1097.png">

**Smartphone (please complete the following information):**

**Testing notes**

- [ ] This bug fix is expected to need manual testing.

**Additional context**

**Comment by RichardPK** [comment]
This is how a space + comma is achieved when CSS is disabled on OD-radio https://github.com/bbc/simorgh/blob/latest/src/app/containers/RadioPageBlocks/Blocks/OnDemandHeading/index.jsx#L68
